### ISSUE TITLE
Is it possible to train for pedestrian and cyclists?

### ISSUE THREAD

**Issue by Abhirami20** [jeasinema/VoxelNet-tensorflow#49]
Can this be used for training Pedestrian and cyclists? In code, it's for car only.

**Comment by Abhirami20** [comment]
@jeasinema
Could you please guide me regarding my doubt?

**Comment by Abhirami20** [comment]
@LinHungShi
Could you please see this issue?

**Comment by LinHungShi** [comment]
No, you can't train all the classes without modifying the code.

**Comment by Abhirami20** [comment]
@LinHungShi  Does preprocessing to all 

> No, you can't train all the classes without modifying the code.
Thank you for your reply!

Does preprocessing effect training on all classes? Now I am working on it but I understood that voxelnet- size required is different for pedestrian/ cyclist and car. So I wants to know if just rewriting code is enough or not?

**Comment by LinHungShi** [comment]
The parameters for car and pedestrian/cyclist are a bit different in the paper. However you can assume they are the same.

**Comment by Abhirami20** [comment]
Thank you so much for your guidance! :)
Is editting config.py enough to train all classes or do I need to edit any other?

**Comment by Abhirami20** [comment]
> The parameters for car and pedestrian/cyclist are a bit different in the paper. However you can assume they are the same.

Thank you so much for the guidance! :)
Is editting config.py enough for training with all classes at once or do I need to edit any other codes?
